source/jormungandr/jormungandr/interfaces/v1/free_floatings.py

```diff
@@ -0,0 +1,105 @@
+# twitter @navitia
+# channel `#navitia` on riot https://riot.im/app/#/room/#navitia:matrix.org
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+from __future__ import absolute_import, print_function, unicode_literals, division
+from flask_restful import abort
+from jormungandr.interfaces.v1.serializer.free_floating import FreeFloatingsSerializer
+from jormungandr import i_manager, timezone, global_autocomplete, authentication
+from jormungandr.interfaces.v1.ResourceUri import ResourceUri
+from jormungandr.interfaces.parsers import default_count_arg_type, places_count_arg_type
+from jormungandr.interfaces.v1.transform_id import transform_id
+from jormungandr.scenarios.utils import free_floatings_type
+from navitiacommon.parser_args_type import OptionValue, CoordFormat
+from jormungandr.instance import Instance
+from typing import Optional, Dict
+
+
+def build_instance_shape(instance):
+    # type: (Instance) -> Optional[Dict]
+    if instance and instance.geojson:
+        return {"type": "Feature", "properties": {}, "geometry": instance.geojson}
+    return None
+
+
+places_types = {
+    'stop_areas',
+    'stop_points',
+    'pois',
+    'addresses',
+    'coords',
+    'places',
+    'coord',
+}  # add admins when possible
+
+
+class FreeFloatingsNearby(ResourceUri):
+    def __init__(self, *args, **kwargs):
+        ResourceUri.__init__(self, *args, **kwargs)
+        parser_get = self.parsers["get"]
+        parser_get.add_argument(
+            "type[]",
+            type=OptionValue(free_floatings_type),
+            action="append",
+            help="Type of free-floating objects to return",
+        )
+        parser_get.add_argument("distance", type=int, default=500, help="Distance range of the query in meters")
+        parser_get.add_argument("count", type=default_count_arg_type, default=10, help="Elements per page")
+        self.parsers['get'].add_argument(
+            "coord",
+            type=CoordFormat(nullable=True),
+            help="Coordinates longitude;latitude used to search " "the objects around this coordinate",
+        )
+
+    def get(self, region=None, lon=None, lat=None, uri=None):
+        self.region = i_manager.get_region(region, lon, lat)
+        timezone.set_request_timezone(self.region)
+        args = self.parsers["get"].parse_args()
+        instance = i_manager.instances.get(self.region)
+
+        # coord in the form of uri:
+        # /coverage/<coverage name>/coord=<lon;lat>/freefloatings_nearby?...
+        if uri:
+            if uri[-1] == '/':
+                uri = uri[:-1]
+            uris = uri.split("/")
+            if len(uris) >= 2 and uris[-2] == 'coord':
+                args["coord"] = transform_id(uris[-1])
+            else:
+                abort(404)
+        # coord as parameter: /coverage/<coverage name>/freefloatings_nearby?coord=<lon;lat>&...
+        else:
+            coord = args.get("coord")
+            if coord is None:
+                abort(404)
```

Comment:
400?

Reply:
This code is already used in api interface files